test/components/FeatureFlag.test.js

```diff
@@ -59,4 +65,13 @@ describe("components/FeatureFlag", () => {
       expect(renderer).toHaveLength(0);
     });
   });
+  it("does render the FeatureFlagRenderer component if forceInitialize is set", () => {
+    const subject = mount(
+      <LaunchDarkly>
+        <FeatureFlag {...defaultProps} forceInitialize />
```

Comment:
How about using `allProps` here, and verifying that `<div>Initial…</div>` renders?

Reply:
*something like `allProps` that is available in this block